.github/workflows/autoSync.yml

```
@@ -0,0 +1,28 @@
+name: Auto Sync
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)